src/scripts/main.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const div = document.createElement('div');
+
+  div.classList.add('notification', type);
+  div.style.top = `${posTop}px`;
+  div.style.right = `${posRight}px`;
+
+  const titles = document.createElement('h2');
+
+  titles.className = 'title';
+  titles.textContent = title;
+  div.appendChild(titles);
+
+  const descriptions = document.createElement('p');
+
+  descriptions.textContent = description;
+  div.appendChild(descriptions);
+
+  document.body.appendChild(div);
+
+  setTimeout(() => {
+    div.style.visibility = 'hidden';
+  }, 2000);
 };
```

Comment:
Consider adding a JSDoc comment to your function to describe its purpose, parameters, and return value. This can make your code easier to understand and maintain.